exchanges/bitstamp/bitstamp_test.go

```diff
@@ -679,35 +686,31 @@ func TestWithdrawInternationalBank(t *testing.T) {
 			IntermediaryBankName:          "Federal Reserve Bank",
 			IntermediaryBankPostalCode:    "2088",
 		},
-		Amount:      -1,
+		Amount:      50,
 		Currency:    currency.USD,
 		Description: "WITHDRAW IT ALL",
 	}
 
-	_, err := b.WithdrawFiatFundsToInternationalBank(context.Background(),
+	w, err := b.WithdrawFiatFundsToInternationalBank(context.Background(),
 		&withdrawFiatRequest)
-	switch {
-	case !sharedtestvalues.AreAPICredentialsSet(b) && err == nil && !mockTests:
-		t.Error("Expecting an error when no keys are set")
-	case sharedtestvalues.AreAPICredentialsSet(b) && err != nil && !mockTests:
-		t.Errorf("Withdraw failed to be placed: %v", err)
-	case mockTests && err == nil:
-		t.Error("Expecting an error until QA pass is completed")
+	if mockTests {
+		assert.NoError(t, err, "WithdrawFiatFundsToInternationalBank should not error")
+		assert.Equal(t, "1", w.ID, "Withdrawal ID should be correct")
+	} else {
+		assert.ErrorContains(t, err, "Check your account balance for details")
```

Comment:
Since this a live test if someone were to set their credentials and canManipulateRealOrders to true, people could have funds in their account but be returned a different error being Satoshi's fake address 😆 Can just add in a standard `assert.NoError(t, err, "WithdrawFiatFunds should not error") similar to `CancelOrder`

Reply:
Amended